documentation/book/appendix_metrics.adoc

```diff
@@ -37,21 +37,30 @@ endif::Kubernetes[]
 
 === Prometheus
 
-The provided Prometheus https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/prometheus.yaml[`prometheus.yaml`] YAML file describes all the resources required by Prometheus in order to effectively monitor a Strimzi Kafka & ZooKeeper cluster.
-These resources lack important production configuration to run a healthy and highly available Prometheus server.
-They should only be used to demonstrate this Grafana dashboard example.
+In Strimzi we use CoreOS Prometheus Operator which can be deployed by applying these https://github.com/coreos/prometheus-operator/tree/master/example/rbac/prometheus-operator[`Prometheus Operator YAML files`] from CoreOS repository.
+This operator allows us to run healthy and highly available Prometheus server what is one of the requirements for the production.
+
+If you use another namespace than in the repository files do you can download and edit single file by using a command.
+
+[source,shell,subs=attributes+]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-deployment.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus-operator-deployment.yaml
+
+Strimzi repository contains configuration files for Prometheus server and by applying them a resources are created.
+These resources are managed by Prometheus Operator.
 
 The following resources are defined:
 
 * A `ClusterRole` that grants permissions to read Prometheus health endpoints of the Kubernetes system, including cAdvisor and kubelet for container metrics.  The Prometheus server configuration uses the Kubernetes service discovery feature in order to discover the pods in the cluster from which it gets metrics.  In order to have this feature working, it is necessary for the service account used for running the Prometheus service pod to have access to the API server to get the pod list.
 * A `ServiceAccount` for the Prometheus pods to run under.
 * A `ClusterRoleBinding` which binds the aforementioned `ClusterRole` to the `ServiceAccount`.
-* A `Deployment` to manage the actual Prometheus server pod.
-* A `ConfigMap` to manage the configuration of Prometheus Server.
-* A `Service` to provide an easy to reference hostname for other services to connect to Prometheus server (such as Grafana).
+* A `Deployment` to manage the Prometheus Operator pod.
+* A `ServiceMonitor` to manage configuration of Prometheus pod.
+* A `Secret` to manage additional Prometheus settings.
+* A `PrometheusRule` to manage alert rules for Prometheus pod.
+* A `Prometheus` to manage Prometheus pod configuration.
```

Comment:
Maybe we should add a link to the Prometheus operator repo where these resources are explained in details.